Dangerfile

```diff
@@ -0,0 +1,25 @@
+def version
+  lib = File.expand_path("../lib", __FILE__)
+  $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+  require 'fastlane/plugin/wpmreleasetoolkit/version'
+  Fastlane::Wpmreleasetoolkit::VERSION
+end
+
+def gemfile_lock_version
+  gemfile_lock = File.read('./Gemfile.lock')
+  gemfile_lock.scan(/fastlane-plugin-wpmreleasetoolkit \((\d+.\d+.\d+)\)/).last.first
+end
+
+# Before checking the version, get rid of any change that `bundle install`
+# might have done.
+`git checkout Gemfile.lock &> /dev/null`
```

Comment:
It's annoying to have to do this, but if `bundle install` run in CI, it will have updated the version in the lockfile, which is exactly what we're trying to figure out.